package.json

```diff
@@ -75,7 +75,7 @@
     "babel-polyfill": "6.26.0",
     "codemirror": "^6.0.0",
     "fast-xml-parser": "^4.0.10",
-    "frontend-components-tinymce-advanced-plugins": "^1.0.2",
+    "frontend-components-tinymce-advanced-plugins": "github:brian-smith-tcril/frontend-components-tinymce-advanced-plugins#build-as-dev-dep-dist",
```

Comment:
will update when https://github.com/openedx/frontend-components-tinymce-advanced-plugins/pull/9 is merged